app/routines/admin/remove_collaborator.rb

```diff
@@ -29,10 +32,12 @@ def exec(publishables:, user:, collaborator_type:)
         end
       end
 
-      outputs.total_count = author_ids.size + copyright_holder_ids.size
+      outputs.total_count += author_ids.size + copyright_holder_ids.size
 
       Author.where(id: author_ids).delete_all unless author_ids.empty?
       CopyrightHolder.where(id: copyright_holder_ids).delete_all unless copyright_holder_ids.empty?
     end
+
+    publishables.respond_to?(:touch_all) ? publishables.touch_all : publishables.each(&:touch)
   end
```

Comment:
`touch_all` is rails 6, right?  Are you just getting ready for it here?

Reply:
Hehe yes